template/$PROJECT_NAME$/apps/$PROJECT_NAME$_api/lib/$PROJECT_NAME$_api/context.ex

```diff
@@ -38,7 +38,7 @@ defmodule <%= @project_name_camel_case %>API.Context do
       ["Bearer " <> token] ->
         %{token: %Token{token: token}}
       _ ->
-        %{}
+        %{token: "invalid"}
```

Comment:
`Accounts.authenticate(%{})` throws an exception. I changed `%{}` to `%{token: "invalid"}` to ensure an error returned by the `Accounts.authenticate`